src/compontents/Button/Button.module.scss

```diff
@@ -0,0 +1,40 @@
+@import "../../variables.scss";
+
+.btn {
+  font-family: $fontFamily;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 28px;
+  text-align: center;
+  height: 56px;
+  border-radius: 28px;
+  min-width: 167px;
+  border: 0;
+  color: $white;
+}
+
+.btn--darkBlue {
```

Comment:
Zastanawiam się czy dobrym pomysłem tutaj nie będzie zastosowanie `@include` dlatego, że jak zapomnisz dodać klasę `btn` to żeby button dalej wyglądał ładnie.